model/flow/header.go

```diff
@@ -104,6 +105,51 @@ func (h *Header) UnmarshalJSON(data []byte) error {
 	return err
 }
 
+// MarshalJSON makes sure the timestamp is encoded in UTC.
```

Comment:
```suggestion
// MarshalCBOR makes sure the timestamp is encoded in UTC.
```

Reply:
@turbolent thanks for the comment! I fixed it and it in its counterpart `UnmarshalCBOR` comment too :-)